server/src/graphql/mutations/unlinkDatabase.ts

```diff
@@ -0,0 +1,68 @@
+import { unlinkDatabaseQueue } from './../../queues/unlinkDatabase';
+import { dbTypeToDokkuPlugin } from './../utils';
+import { MutationResolvers } from '../../generated/graphql';
+
+import { prisma } from '../../prisma';
+
+export const unlinkDatabase: MutationResolvers['unlinkDatabase'] = async (
+  _,
+  { input },
+  { userId }
+) => {
+  if (!userId) {
+    throw new Error('Unauthorized');
+  }
+
+  const { databaseId, appId } = input;
+
+  const [database, app] = await Promise.all([
+    prisma.database.findOne({
+      where: {
+        id: databaseId,
+      },
+      include: {
+        apps: {
+          where: {
+            id: appId,
+          },
+        },
+      },
+    }),
+    prisma.app.findOne({
```

Comment:
do you need to do this find as the app is already included in the previous query?

Reply:
@pradel no, not really. Fixed!